hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionConfiguration.java

```diff
@@ -31,6 +31,7 @@
 import static org.apache.hadoop.hbase.HConstants.HBASE_CLIENT_OPERATION_TIMEOUT;
 import static org.apache.hadoop.hbase.HConstants.HBASE_CLIENT_PAUSE;
 import static org.apache.hadoop.hbase.HConstants.HBASE_CLIENT_PAUSE_FOR_CQTBE;
+import static org.apache.hadoop.hbase.HConstants.HBASE_CLIENT_PAUSE_FOR_SERVER_OVERLOADED;
```

Comment:
This PR predates my discussion around constants. I added this in HConstants because that's where the existing CQTBE is. It seems like we've come to a conclusion between those who have commented on the discussion thread, so I could update this based on that new convention (still need to do the other followups from that thread)

Reply:
My next commit will include moving this to the ConnectionConfiguration in hbase-client, along with making that class LP(CONFIG). If we'd prefer to leave the LP(CONFIG) change for the forthcoming jira from my discussion thread, that's ok too.